p8e-api/build.gradle

```diff
@@ -157,3 +181,41 @@ dependencyManagement {
         cacheChangingModulesFor 0, 'seconds'
     }
 }
+
+task downloadZipFile(type: Download) {
```

Comment:
hmm, good point. I'll do some tests/investigation to see what happens when there is no internet connection when building. Could maybe still leave it as an independent Gradle task and remove the generateProtoTasks dependency on it, and have it copy the protobufs into committed source instead of the build directory, so we can just run it whenever we need to update

Reply:
That's a good idea.

Reply:
I have this going now, as well as pulling in the base Cosmos SDK protos (the Gradle equivalent of this snippet Phil shared for getting the base cosmos protos `curl -L https://github.com/cosmos/cosmos-sdk/tarball/$(go list -mod=mod -m github.com/cosmos/cosmos-sdk | cut -d' ' -f2) | tar xvf - --strip-components 1 --include '*.proto'`), I think this has the base services that we will need for the full grpc switchover once we are able to develop against the new chain